easybuild/tools/job/gc3pie.py

```diff
@@ -260,3 +270,12 @@ def _print_status_report(self):
         states = ', '.join(["%d %s" % (stats[s], s.lower()) for s in stats if s != 'total' and stats[s]])
         print_msg("GC3Pie job overview: %s (total: %s)" % (states, self.job_cnt),
                   log=self.log, silent=build_option('silent'))
+        if __debug__:
```

Comment:
Since `eb` runs `python -O -m easybuild.main`, this is never going to pass?

Maybe you want to use `if build_option('debug'):`?